src/main/scala/com/greencatsoft/angularjs/AngularExecutionContext.scala

```diff
@@ -20,7 +20,7 @@ class AngularExecutionContext(
       case t: Throwable => reportFailure(t)
     }
 
-    timeout(run _, 0)
```

Comment:
This should work, but borks on scala 2.12 unfortunately. probably a compiler bug. trying to repro and report.